packages/vega-scenegraph/src/SVGRenderer.js

```diff
@@ -529,6 +540,20 @@ prototype._update = function(mdef, el, item) {
   // apply svg attributes
   mdef.attr(emit, item, this);
 
+  // optimization: if ariaHidden is true, we can remove other aria properties
+  if (item.ariaHidden) {
```

Comment:
Is this necessary? `delete` can be a relatively costly operation. Another option might be to set these to null or undefined if it is necessary to suppress the values.

Reply:
On second thought, this might be buggy. What if ariaHidden is a signal defined in the update set, when all the other properties are constants in the enter set? I think that could result in inaccurate output here. We may need to condition lookup of other aria properties on the current value of ariaHidden.

Reply:
I'm happy to just remove this. I assumed that there would be value in removing things from the generated SVG.